### PR TITLE
Improved test code coverage for plugin/pkg/admission/admit.

### PR DESCRIPTION
**Pull request by gyliu513**
**What this PR does / why we need it**:'
part of #39559 , code coverage improved from 60% to 80%

### CODE REVIEW

**Comment by gyliu513** [comment]
Also part of #40782

File: plugin/pkg/admission/admit/admission_test.go [hunk 1]
```diff
@@ -18,12 +18,26 @@ package admit
...
 	"testing"
+
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/kubernetes/pkg/api"
 )
 
 func TestAdmission(t *testing.T) {
 	handler := NewAlwaysAdmit()
-	err := handler.Admit(nil)
+	err := handler.Admit(admission.NewAttributesRecord(nil, nil, api.Kind("kind").WithVersion("version"), "namespace", "name", api.Resource("resource").WithVersion("version"), "subresource", admission.Create, nil))
```
Comment (k82cn) [inline plugin/pkg/admission/admit/admission_test.go]: Both nil & un-nil should test.

**Review by k82cn — COMMENTED** [review]
Should `BUILD` be updated?

File: plugin/pkg/admission/admit/admission_test.go (hunk 1 above)
Comment (gyliu513) [inline plugin/pkg/admission/admit/admission_test.go]: Done @k82cn

**Comment by gyliu513** [comment]
@k82cn why update `BUILD`? I did not introduce any new file here.

**Comment by k82cn** [comment]
`BUILD` for new imported package.

**Comment by derekwaynecarr** [comment]
@k8s-bot ok to test.

**Comment by derekwaynecarr** [comment]
i applaud the goal of increased test coverage.  i also think this particular admission plug-in is a good candidate for deprecation.  it was originally authored as an example for how to write an admission plug-in in the early days of the project, and at this point, there is no shortage of these things that an example is less valuable.

that said /lgtm

**Comment by gyliu513** [comment]
@derekwaynecarr do you know why this does not get committed after approved for days? Thanks!

**Comment by k82cn** [comment]
@k8s-bot ok to test.

**Comment by gyliu513** [comment]
@k8s-bot ok to test.

**Comment by gyliu513** [comment]
@derekwaynecarr ping, do you know why this does not get committed after approved for days? Thanks!

**Comment by derekwaynecarr** [comment]
@gyliu513 -- i dont... let me try again.

**Comment by derekwaynecarr** [comment]
@k8s-bot ok to test.

**Comment by derekwaynecarr** [comment]
@k8s-bot test this

**Comment by derekwaynecarr** [comment]
this pr is fine, but i don't think its required for 1.6 milestone.  this will merge once the 1.7 queue is opened up.  thanks again for the contribution.

**Comment by gyliu513** [comment]
Thanks @derekmahar , can you please show some detail how to make sure this can be merge for 1.7? Just curious that sometimes, even if the PR was approved, it still does not got merged after days, so want to get some detail for this. Thanks!

**Comment by k82cn** [comment]
>  can you please show some detail how to make sure this can be merge for 1.7?

The submit queue of 1.7 is closed for 1.6 release. Please check merging pipeline here: http://submit-queue.k8s.io/#/queue . It'll merge 10 ~ 20 lgtm/approved PR per day on average, as it will re-run all test before merged.

**Comment by k82cn** [comment]
@k8s-bot cvm gce e2e test this
